turbopack/crates/turbopack-node/js/src/transforms/webpack-loaders.ts

```diff
@@ -19,27 +19,17 @@ import { type StructuredError } from "src/ipc";
 
 export type IpcInfoMessage =
   | {
-    type: "fileDependency";
-    path: string;
-  }
-  | {
-    type: "buildDependency";
-    path: string;
-  }
-  | {
-    type: "dirDependency";
-    path: string;
-    glob: string;
-  }
-  | {
-    type: "envDependency";
-    name: string;
-  }
+      type: 'dependencies'
+      envVariables?: string[]
+      directories?: Array<[string, string]>
+      filePaths?: string[],
+      build?: string[],
```

Comment:
not: maybe call it `files` for consistency?

Reply:
renamed to buildFilePaths